src/decoders/router_decoder_custom_sup.erl

```diff
@@ -91,9 +91,18 @@ add(Decoder) ->
     add(ID, Function).
 
 -spec delete(binary()) -> ok.
+-ifdef(TEST).
```

Comment:
Do we still need this test wrapper for `delete/1`?
I ran all eunits, and all the changed CTs without failure

Reply:
Fixed in commit f814c72